### PR TITLE
Self Nomination for Tyler Yahn to the GC

### PR DESCRIPTION
**Pull request by MrAlias**

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]
+1

**Comment by bhs** [comment]
@MrAlias thanks for the PR! Consider yourself nominated.

Before merging this PR, you need to be "ratified": the process is described [here](https://github.com/open-telemetry/community/blob/master/governance-charter.md#eligibility-for-candidacy). TL;DR:

>To be ratified as a candidate, the nominee must accept the nomination and three Members of Standing (including the nominator, if she/he has standing) from three different employers, must endorse the nomination.

... So please just have three OTel "members of standing" endorse your nomination by commenting as such on this PR.

We will count @Aneurysm9's `+1` above as one of those endorsements.

File: elections/2020/governance-committee-candidates.md [hunk 1]
```diff
@@ -25,3 +26,20 @@ occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim
 id est laborum.
 
 ---
+
+### Tyler Yahn (He/Him)
+
+![Tyler Yahn](https://media-exp1.licdn.com/dms/image/C4E03AQGIVMZfZIio8Q/profile-displayphoto-shrink_200_200/0?e=1608163200&v=beta&t=yFPwjgZomuLSexgC3aZ2H9Lzo1ZXsLNAaqz8lIns1do)
```
Comment (SergeyKanzhelev) [inline elections/2020/governance-committee-candidates.md:32]: nit: I'd suggest to include image to this repo instead of an external link

**Comment by mtwo** [comment]
+1

**Comment by pyohannes** [comment]
+1

**Comment by kolanos** [comment]
+1

**Review by mtwo — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Comment by cxhansen** [comment]
+1. I think Tyler would make an excellent GC member. He cares deeply about OpenTelemetry and its community.

**Comment by jmacd** [comment]
:100: Tyler has my endorsement!

**Pull request merged**